lib/daffy_lib/models/encryption_key.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Just a note but when you deploy this on roadrunner you'll have to migrate all existing EncryptionKeys to this table

Reply:
Roadrunner looks like its migration to add CreateEncryptionKeys is identical to what's currently in the Wile PR using libraries?  I'm not sure I follow what needs to be done.